src/clientsettingsdlg.cpp

```diff
@@ -229,12 +232,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                          "let you choose input or outputs of your device(s). "
                                          "More information can be found on jamulus.io." );
 
-    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: " ) + APP_NAME +
-                                      tr ( " currently only supports devices supporting a sample rate of " ) +
-                                      QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) +
-                                      tr ( "Hz. "
-                                           "You will not be able to select a driver/device which doesn't. "
-                                           "For more help see jamulus.io." ) +
+    QString strSndCardDriverSetupTT = QString ( tr ( "Opens the driver settings. Note: %1 currently only supports devices "
+                                                     "supporting a sample rate of %2 Hz. "
+                                                     "You will not be able to select a driver/device which doesn't. "
```

Comment:
That statement seems to be redundant. 
```suggestion

```

Reply:
This would make it clearer. IMHO it's not redundant.

Reply:
OK sure, it's only words I guess.